src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar__start-day--mon calendar__month-lenght--31">
```

Comment:
```suggestion
    <div class="calendar calendar__start-day--mon calendar__month-length--31">
```
now calendar not work properly for month-length modifier.
take a look on screen. when modifier with 28 should be visible 28 days on calendar
![Screenshot from 2023-10-31 15-35-15](https://github.com/mate-academy/layout_calendar/assets/33029864/a9e611f2-b569-4dbd-abb0-37815800c868)